### PR TITLE
Added a note for format query param to openapi guide

### PR DESCRIPTION
**Pull request by evanshortiss**
This is a minor addition to the OpenAPI and Swagger guide. I needed the OpenAPI spec in JSON format instead of YAML, and wasn't sure how to request it.

PR result:

Screenshot 2021-08-20 at 10 18 11 AM

### CODE REVIEW

File: docs/src/main/asciidoc/openapi-swaggerui.adoc [hunk 1]
```diff
@@ -260,6 +260,16 @@ quarkus.smallrye-openapi.path=/swagger
...
++[NOTE]
++====
++You can request the OpenAPI in JSON format using the `format` query parameter. For example:
++[source, properties]
++----
++/q/openapi?format=json
++----
++====
++
+
```
Comment (gsmet) [inline docs/src/main/asciidoc/openapi-swaggerui.adoc:272]: Not sure why you added `+` at the beginning of each line?
```suggestion
[NOTE]
====
You can request the OpenAPI in JSON format using the `format` query parameter. For example:
[source, properties]
----
/q/openapi?format=json
----
====

```

**Review by gsmet — CHANGES_REQUESTED** [review]
Thanks, nice addition.

Can you have a look at my suggestion? And squash everything once done? Thanks!

File: docs/src/main/asciidoc/openapi-swaggerui.adoc (hunk 1 above)
Comment (evanshortiss) [reply docs/src/main/asciidoc/openapi-swaggerui.adoc]: Oh wow, copy paste error. I originally cloned and changed the "guides" repo and copied the changes  🤦 

Fixed now.

**Review by gsmet — APPROVED** [review]

**Comment by gsmet** [comment]
Much better, thanks :).

**Pull request merged**